common/components/WalletDecrypt/Mnemonic.jsx

```diff
@@ -1,27 +1,129 @@
 import React, { Component } from 'react';
-import translate from 'translations';
+import translate, { translateRaw } from 'translations';
+import { validateMnemonic, mnemonicToSeed } from 'bip39';
+
+import DeterministicWalletsModal from './DeterministicWalletsModal';
+
+import DPATHS from 'config/dpaths.json';
+const DEFAULT_PATH = DPATHS.MNEMONIC[0].value;
+
+type State = {
+  phrase: string,
+  pass: string,
+  seed: string,
+  dPath: string
+};
 
 export default class MnemonicDecrypt extends Component {
+  props: { onUnlock: any => void };
+  state: State = {
+    phrase: '',
+    pass: '',
+    seed: '',
+    dPath: DEFAULT_PATH
+  };
+
   render() {
+    const { phrase, seed, dPath, pass } = this.state;
+    const isValidMnemonic = validateMnemonic(phrase);
+
     return (
       <section className="col-md-4 col-sm-6">
-        <div id="selectedUploadKey">
-          <h4>{translate('ADD_Radio_2_alt')}</h4>
-
+        <div id="selectedTypeKey">
+          <h4>
+            {translate('ADD_Radio_5')}
+          </h4>
+          <div className="form-group">
+            <textarea
+              id="aria-private-key"
+              className={`form-control ${isValidMnemonic
+                ? 'is-valid'
+                : 'is-invalid'}`}
+              value={phrase}
+              onChange={this.onMnemonicChange}
+              placeholder={translateRaw('x_Mnemonic')}
```

Comment:
You can save the `translateRaw` import by passing `true` as the argument for the `textOnly` parameter:

`translate('x_Mnemonic', true)`

Reply:
Makes me question if `translateRaw` should even be exported.